### PR TITLE
Add line chart pattern

### PR DESCRIPTION
**Pull request by serenamarie125**
Add the pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
@LHinson this one is ok to merge, but the images are huge and they will not look good on the website unless we resize them. Maybe we can merge them for now and then create a story to resize all the wrong images ?

**Comment by LHinson** [comment]
+1 @andresgalante I've made note to add a couple stories for that effort.

File: pattern-library/data-visualization/line-chart/design/overview.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Pattern Name Goes Here
+
+## Overview
+The most common use case for line charts is to compare several data sets, or to show data over a period of time. As a rule of thumb, line charts are a great way to show continuous data over time. If you want to show and compare categorical data, you may consider using a bar chart.
+
+Multiple lines on the same chart allow the user to visualize relationships between varying data sets, such as correlated events, similarities or unexpected differences. We recommend displaying no more than 6 lines on a single graph to avoid confusion.
+
+Jump to [Line Chart](../pattern-library/data-visualization/line-chart/#example-overview-1) or [Single Line Chart](../pattern-library/data-visualization/line-chart/#example-overview-2)
```
Comment (LHinson) [inline pattern-library/data-visualization/line-chart/design/overview.md]: Links do not work.

**Comment by serenamarie125** [comment]
@LHinson should be all set now.  LMK if there is anything else that needs to be addressed!

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**